src/lib/openapi/spec/public-signup-token-schema.ts

```diff
@@ -18,28 +18,40 @@ export const publicSignupTokenSchema = {
     ],
     properties: {
         secret: {
+            description: 'The token',
             type: 'string',
+            example: 'a3c84b25409ea8ca1782ef17f94a42fc',
         },
         url: {
             description:
                 'The public signup link for the token. Users who follow this link will be taken to a signup page where they can create an Unleash user.',
             type: 'string',
+            example:
+                'https://sandbox.getunleash.io/enterprise/new-user?invite=a3c84b25409ea8ca1782ef17f94a42fc',
         },
         name: {
+            description: "The token's name",
             type: 'string',
+            example: 'default',
         },
         enabled: {
+            description: 'Whether the token is active',
```

Comment:
Can a token be active **and** past its expiration date? Or is that an invalid state? If so, maybe we could say something like this?

```suggestion
            description: 'Whether the token is active. This property will always be `false` for a token that has expired.',
```